frontend/src/utilities/useWatchDashboardConfig.tsx

```diff
@@ -4,39 +4,14 @@ import { POLL_INTERVAL } from './const';
 import { useDeepCompareMemoize } from './useDeepCompareMemoize';
 import { fetchDashboardConfig } from '../services/dashboardConfigService';
 
-export const blankDashboardCR: DashboardConfig = {
-  apiVersion: 'opendatahub.io/v1alpha',
-  kind: 'OdhDashboardConfig',
-  metadata: {
-    name: 'odh-dashboard-config',
-  },
-  spec: {
-    dashboardConfig: {
-      enablement: true,
-      disableInfo: false,
-      disableSupport: false,
-      disableClusterManager: false,
-      disableTracking: false,
-      disableBYONImageStream: false,
-      disableISVBadges: false,
-      disableAppLauncher: false,
-      disableUserManagement: false,
-    },
-    groupsConfig: {
-      adminGroups: 'odh-admins',
-      allowedGroups: 'system:authenticated',
-    },
-  },
-};
-
 export const useWatchDashboardConfig = (): {
```

Comment:
We should probably move this utility to the `App` folder right? Maybe even call it something like `useApplicationSettings` or something when we move it. That way no one will get wise and try to import it elsewhere. We want to favour people into using the app context one, so if we rename it might make it even less palpable.